dkist/io/file_manager.py

```diff
@@ -245,29 +266,28 @@ def download(self, path="/~/", destination_endpoint=None, progress=True):
 
         inv = self._ndcube.meta["inventory"]
 
-        base_path = Path(DKIST_DATA_CENTRE_DATASET_PATH.format(**inv))
-        # TODO: Default path to the config file
-        destination_path = Path(path) / inv['primaryProposalId'] / inv['datasetId']
+        base_path = Path(net_conf.dataset_path.format(**inv))
+        destination_path = path or self.basepath or "/~/"
+        destination_path = Path(destination_path).as_posix()
+        destination_path = Path(destination_path.format(**inv))
 
+        # TODO: If we are transferring the whole dataset then we should use the
+        # directory not the list of all the files in it.
         file_list = [base_path / fn for fn in self.filenames]
         file_list.append(Path("/") / inv['bucket'] / inv['asdfObjectKey'])
```

Comment:
There is a browse movie and quality report in the same place as the asdf.  I concur with the directory transfer TODO, but in the mean time you might want to consider appending those to the file list like you do the asdf.